scripts/make/cargo.mk

```diff
@@ -17,7 +17,7 @@ build_args := \
   $(build_args-$(MODE)) \
   $(verbose)
 
-RUSTFLAGS := -C link-arg=-T$(LD_SCRIPT) -C link-arg=-no-pie
+RUSTFLAGS := -C link-arg=-T$(LD_SCRIPT) -C link-arg=-no-pie -C link-arg=-znostart-stop-gc
```

Comment:
I this place need an annotation, like add refs about what is 'nostart-stop-gc', I find something here: https://releases.llvm.org/14.0.0/tools/lld/docs/ELF/start-stop-gc.html